src/main/java/edu/harvard/iq/dataverse/authorization/AuthenticationServiceBean.java

```diff
@@ -334,18 +353,16 @@ public AuthenticatedUser authenticate( String authenticationProviderId, Authenti
                 user = userService.updateLastLogin(user);
             }
             
-            /**
-             * @todo Why does a method called "authenticate" have the potential
-             * to call "createAuthenticatedUser"? Isn't the creation of a user a
-             * different action than authenticating?
-             *
-             * @todo Wouldn't this be more readable with if/else rather than
-             * ternary?  (please)
```

Comment:
I like the new version without the ternary operator. Thanks.

Reply:
I might have been doing LISP at the time. I don't remember much 😜